### PR TITLE
Show messages from customGas.js in Send Offline page

### PR DESCRIPTION
**Pull request by domalaq**

### CODE REVIEW

File: app/scripts/controllers/offlineTxCtrl.js [hunk 1]
```diff
@@ -97,8 +97,16 @@ var offlineTxCtrl = function($scope, $sce, walletService) {
...
     }
     $scope.validateAddress = function(address, status) {
+        $scope.customGasMsg = ''
+        
         if (ethFuncs.validateEtherAddress(address)) {
             $scope.notifier.info(globalFuncs.successMsgs[0]);
+
+            for (var i in CustomGasMessages) {
+                if ($scope.tx.to.toLowerCase() == CustomGasMessages[i].to.toLowerCase()) {
+                    $scope.customGasMsg = CustomGasMessages[i].msg != '' ? CustomGasMessages[i].msg : ''
```
Comment (crptm) [inline app/scripts/controllers/offlineTxCtrl.js:107]: This does nothing

**Comment by domalaq** [comment]
hi @crptm. Why do you think that?It shows some custom message depending on entered address. You can check this address filling it in To field at Send Offline page for example:0xc9f32ce1127e44c51cbd182d6364f3d707fd0d47Sent from Nylas Mail, the best free email app for work
          
            
            On Jun 23 2017, at 4:46 pm, crptm <notifications@github.com> wrote:
            
            
              @crptm commented on this pull request.



In app/scripts/controllers/offlineTxCtrl.js:
>          if (ethFuncs.validateEtherAddress(address)) {
             $scope.notifier.info(globalFuncs.successMsgs[0]);
+
+            for (var i in CustomGasMessages) {
+                if ($scope.tx.to.toLowerCase() == CustomGasMessages[i].to.toLowerCase()) {
+                    $scope.customGasMsg = CustomGasMessages[i].msg != '' ? CustomGasMessages[i].msg : ''

This does nothing

**Comment by tayvano** [comment]
@domalak ETH Address, please :)

**Comment by tayvano** [comment]
@domalak ETH Address, please :)

**Comment by crptm** [comment]
@domalak if value is not an empty string, use it; if value is empty string use empty string. This should always result in same value, right?

**Comment by goppu** [comment]
Storiqa tokens coin missing later fewday please send me massage

**Pull request merged**